editor-std/ui_data_es.txt

```diff
@@ -97,17 +97,17 @@ Wiring Tools = Herramientas de Cableado
 Power Objects = Objetos de Potencia
 PCB Tools = Herramientas PCB
 Footprint Tools = 
-Layers Tool = 
-Layer Manager = 
-Enable and Show All Layers = 
+Layers Tool = Herramientas de capa
+Layer Manager = Administrador de capa
```

Comment:
Hi, Is the Manager should not translate to Administador? it isn't mean the administor, it is like the Footpirnt Manager, Design Manager